srcpkgs/asahi-base/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'asahi-base'
+pkgname=asahi-base
+version=20241001
+revision=1
+archs="aarch64*"
+build_style=meta
+depends="linux-asahi m1n1"
```

Comment:
```suggestion
depends="linux-asahi m1n1 asahi-firmware asahi-uboot"
```

Reply:
I think the readme is clear about that:
```
At the very least, install base-system, asahi-base and an initram
generator.  By default, also asahi-uboot, grub-arm64-efi and asahi-scripts
are required, unless one changes initram or payload for m1n1.
```
The asahi-firmware is a dependency of asahi-scripts.
I don't like to force those two into asahi-base, as e.g. one can use m1n1 without uboot,
and with tinyramfs you don't even need asahi-scripts, strictly speaking.
For the same reason, I'm not adding dracut as dependency.